core/domain/prod_validators.py

```diff
@@ -1136,20 +1136,17 @@ class ExplorationRightsModelValidator(base_model_validators.BaseModelValidator):
 
     @classmethod
     def _get_external_id_relationships(cls, item):
-        cloned_from_exploration_id = []
-        if item.cloned_from:
-            cloned_from_exploration_id.append(item.cloned_from)
         snapshot_model_ids = [
             '%s-%d' % (item.id, version)
             for version in python_utils.RANGE(1, item.version + 1)]
         return [
+            # Note: There can be a cloned from exploration id in the rights
```

Comment:
Actually I have a different suggestion. Let's just delete all explorations where cloned_from is True. We don't really support that use case anyway, and we dropped that functionality pretty early on.

Reply:
Hmm, this will require more infra - deleting an exploration will require that we remove all related external models otherwise new errors creep in. So, I am reverting change as of now and have filed an issue to safely deprecate cloned_from field: https://github.com/oppia/oppia/issues/10828.